test/Microsoft.Health.Dicom.Tests.Integration/Persistence/SqlIndexDataStoreTestHelper.cs

```diff
@@ -16,6 +18,15 @@ public class SqlIndexDataStoreTestHelper : IIndexDataStoreTestHelper
     {
         private readonly string _connectionString;
 
+        private static readonly IReadOnlyDictionary<CustomTagDataType, string> TableNames = new Dictionary<CustomTagDataType, string>()
+            {
```

Comment:
Sorry, I meant that the original mapping of this is relevant. Can it be a static mapping that is used in both test code and production code?

---
In reply to: [590904575](https://github.com/microsoft/dicom-server/pull/612#discussion_r590904575) [](ancestors = 590904575)

Reply:
Hi Sneha, I didn't find where production code use it, could you point me out?
I will make it static

Reply:
AddInstanceTableValuedParametersBuilder uses this in a hardcoded manner since it maps StringData -> InsertStringCustomTagTableTypeV1Row which is then mapped in the sql code to the CustomTagString table. It might be useful to encode it a more clear configuration based mapping.

---
In reply to: [591963573](https://github.com/microsoft/dicom-server/pull/612#discussion_r591963573) [](ancestors = 591963573)